📝 Clarify single-command vs multi-command behaviour in README (#1268)
fastapi/typer: README.md

* Add two subcommands with `@app.command()`.
* Execute the `app()` itself, as if it was a function (instead of `typer.run`).

> 
> >**Note on Single vs Multi-command mode**:
> > If your app only has one command, Typer will automatically enter "single-command mode", meaning the command name is **omitted** in usage (`python main.py Camila`).
> > However, once you define multiple commands, you must **explicitly include the command name** (`python main.py hello Camila`).
> > This behavior can be surprising — see [One or Multiple Commands](https://typer.tiangolo.com/tutorial/commands/one-or-multiple/) for more details.
> 

Reviewer: Attempt to shorten (and some rephrasing):

```suggestion

>**Note**:
> If your app only has one command, the command name is **omitted** in usage: `python main.py Camila`.
> However, when there are multiple commands, you must **explicitly include the command name**: `python main.py hello Camila`.
> See [One or Multiple Commands](https://typer.tiangolo.com/tutorial/commands/one-or-multiple/) for more details.

```
